### PR TITLE
added support ticket routing

### PR DESCRIPTION
**Pull request by brenkao**

### CODE REVIEW

File: docs/cookbook/support_ticket_routing.md [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+    - [Prompts](../learn/prompts.md)
+    - [Calls](../learn/calls.md)
+    - [Tools](../learn/tools.md)
+    - [Dynamic Configuration](../learn/dynamic_configuration.md)
+    - [Chaining](../learn/chaining.md)
+    - [Response Models](../learn/response_models.md)
+
+!!! note "Background"
+    Traditional machine learning techniques like text classification were previously used to solve routing. LLMs have enhanced routing by being able to better interpret nuances of inquiries as well as using client history and knowledge of the product to make more informed decisions.
```
Comment (willbakst) [inline docs/cookbook/support_ticket_routing.md:15]: nit: missing extra newline?

File: docs/cookbook/support_ticket_routing.md [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+
+## Imitating a Company’s Database/Functionality
+
+!!! Note
+
+For both privacy and functionality purposes, these data types and functions in *no way* represent how a company’s API should actually look like. However, extrapolate on these gross oversimplifications to see how the LLM would interact with the company’s API.
```
Comment (willbakst) [inline docs/cookbook/support_ticket_routing.md:21]: nit: missing indent?

File: docs/cookbook/support_ticket_routing.md [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+    - [Dynamic Configuration](../learn/dynamic_configuration.md)
+    - [Chaining](../learn/chaining.md)
+    - [Response Models](../learn/response_models.md)
+
+!!! note "Background"
+    Traditional machine learning techniques like text classification were previously used to solve routing. LLMs have enhanced routing by being able to better interpret nuances of inquiries as well as using client history and knowledge of the product to make more informed decisions.
+
+## Imitating a Company’s Database/Functionality
+
+!!! Note
```
Comment (willbakst) [inline docs/cookbook/support_ticket_routing.md:19]: nit: should be "note", also what about a title? or no title?

Comment (brenkao) [reply docs/cookbook/support_ticket_routing.md:19]: no title i think?

Comment (willbakst) [reply docs/cookbook/support_ticket_routing.md:20]: ok, still update then to `!!! note ""`

**Review by willbakst — APPROVED** [review]

**Pull request merged**